core/src/main/java/com/linecorp/armeria/server/annotation/Attribute.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.linecorp.armeria.common.RequestContext;
+import com.linecorp.armeria.internal.server.annotation.DefaultValues;
+
+import io.netty.util.AttributeKey;
+
+/**
+ * Annotation for mapping a {@link AttributeKey} from {@link RequestContext} onto the following elements.
+ *
+ * <p>a parameter of an annotated service method. </p>
```

Comment:
Maybe remove this?

Reply:
<img width="682" alt="image" src="https://github.com/line/armeria/assets/90125071/15abb843-89cc-47d6-9841-dbba68e73cc6">
I removed `<p>a parameter of an annotated service method. </p>`.